reconcile/utils/oc_map.py

```diff
@@ -122,7 +122,8 @@ def _init_oc_client(
                     connection_parameters=connection_parameters
                 )
             try:
-                oc_client: OCCli | OCLogMsg = self._oc_cls(
+                # TODO: fix ignore type
+                oc_client: OCCli | OCLogMsg = self._oc_cls(  # type: ignore
```

Comment:
I honestly don't know why this is failing now... ignoring it for the moment, this should be addressed in a separate PR.

Reply:
related to https://github.com/app-sre/qontract-reconcile/pull/4728